docs/modules/plugins/pages/plugin-visual.adoc

```diff
@@ -209,6 +209,25 @@ image::example.png[Difference between native/web footer/header image]
 
 === Mobile Native Applications
 
+==== Properties
+
+[cols="3,1,1,3", options="header"]
+|===
+|Property Name
+|Acceptable values
+|Default
+|Description
+
+|`mobile.screenshot.downscale`
+|`boolean`
+|`true`
+|Downscale checkpoint image according to a device DPR value
+|===
+
+Please see the image to get a clue about difference between original and downscaled images.
+
+image::origininal_and_downscaled_checkpoints.png[Difference between original and dpr downscaled image]
```

Comment:
add more details around the image: what is the original image, what is the downscaled image, what is DPR (please bear in mind it's not obvious for users)